superset/templates/superset/basic.html

```diff
@@ -104,5 +104,8 @@ <h4 data-test="modal-title" class="modal-title"></h4>
         {{ js_bundle(entry) }}
       {% endif %}
     {% endblock %}
+    {% block custom_template %}
```

Comment:
Can we just move the `include` into `block tail_js` like in `base.html`? I'm not sure why this file need a new block but the other one does not.

Reply:
oh right, this is because we have one template https://github.com/preset-io/incubator-superset/blob/0e31413fa36180a74febe6a17b00072907d3d62d/superset/templates/superset/crud_views.html#L24 that is overriding tail_js without calling super(). I think it has to do with the menu already being included in that "app". I guess we could add the `{% include "tail_js_custom_extra.html" %}` call in there too.